src/internal.h

```diff
@@ -552,11 +552,14 @@ struct _GLFWwindow
 
     // Preedit texts
     unsigned int*       preeditText;
-    int                 ntext;
-    int                 ctext;
-    int*                preeditAttributeBlocks;
-    int                 nblocks;
-    int                 cblocks;
+    int                 preeditLength;
+    int                 preeditLengthAllocated;
```

Comment:
I prefer `preeditBufferCount`.
* `Length` or `Count` or something similar suffix should be used for variables which stores array length
* GLFW seems prefer `Count` rather than `Length`

Reply:
Thanks for the feedback!

I think `Buffer` is better than `Allocated` too! Thanks! I will fix it!

> Length or Count or something similar suffix should be used for variables which stores array length

日本語ですみません、`Length`が`Count`などのサフィックスは配列の長さを保持する変数に使うべきということですよね。
その趣旨を理解出来ていない気がしたので確認させてほしいです。
今回でいうと、配列の長さを表す変数ではないものに`Length`や`Count`を使ってしまっているので直した方が良い、ということでしょうか？
`preeditLength`も`preeditLengthAllocated`も、どちらも配列の長さですかね？その考え方がおかしいですか？
(前者は終端文字までの長さ、後者はメモリ領域を確保済みの長さ、というのは理解しています)

Reply:
配列の長さを表す変数の命名規則は統一した方が良くて、特にサフィックスは統一した方が良いんじゃね？という意図でした。preeditLengthAllocatedはそれ（配列の長さを表す変数）だと思うのですが、`Allocated`が付いたことでサフィックスが`Length`じゃなくなって命名規則が壊れてるように思えたので。
またGLFWの他のコードだとサフィックスは`Count`使ってそうじゃない？ということでした。

Reply:
なるほど！`Allocated`が末尾に付くことでサフィックスが`Length`ではなくなった、ということですか！理解しました！
ありがとうございます！

実は、こういう形容詞を最後に付けることがサフィックスの統一性を崩すという概念があまり無かった(形容詞が最後についてもサフィックスは`Length`のままだと認識するクセがあった)のですが、自分の勝手なクセだったかもしれません。
考え方が広がりました。

Reply:
I will fix `preeditLength` to `preeditCount` too!

Reply:
Hmm, maybe `preeditTextCount` is better

Reply:
なんとなく文字列という認識があって`Count`にしにくい感覚があったような気がした。
しかし改めて考えると`Count`に揃えたほうが間違いなくスッキリしそう